backend/handler/handler.go

```diff
@@ -32,8 +36,9 @@ type InitializeResponse struct {
 }
 
 type registerRequest struct {
-	Name     string `json:"name"`
-	Password string `json:"password"`
+	// name and password are set be aplhanumeric only
```

Comment:
I feel like in a real world application we should also allow for special characters-- but its probably fine if its just alphanumeric XD

Reply:
Will updated this later.